CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## [5.0.0](https://github.com/plivo/plivo-java/tree/v5.0.0) (2021-07-05)
+- **BREAKING**: getTotalCount() method removed for list MDR.
+- Moved get() method to PowerPackLister
```

Comment:
The moving of get() part - Should this be communicated in changelog? IOW, will this change break existing users? If not, we can remove it.